internal/factsengine/gatherers/sbddump_test.go

```diff
@@ -0,0 +1,139 @@
+package gatherers
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	mocks "github.com/trento-project/agent/internal/factsengine/gatherers/mocks"
+)
+
+type SBDDumpTestSuite struct {
+	suite.Suite
+}
+
+func TestSBDDumpTestSuite(t *testing.T) {
+	suite.Run(t, new(SBDDumpTestSuite))
+}
+
+func (suite *SBDDumpTestSuite) TestSBDDumpGathererMissingFact() {
+	mockExecutor := new(mocks.CommandExecutor)
+
+	mockOutputFile, _ := os.Open("../../../test/fixtures/gatherers/sbddump.output")
```

Comment:
If we are using the same dump in all the tests, we could have this in the test suite, and store it in a variable, so we don't need to open the file every check

Reply:
If it's ok, let's just keep it like this as I will make changes around this next week when I add DI for the CommandExecutor